package.json

```diff
@@ -36,11 +36,15 @@
     "start": "echo 'Use pnpm dev for web client and pnpm dev:sync for sync worker in separate terminals'"
   },
   "dependencies": {
+    "@codemirror/autocomplete": "^6.18.6",
     "@codemirror/basic-setup": "^0.20.0",
     "@codemirror/commands": "^6.8.1",
     "@codemirror/lang-markdown": "^6.3.3",
     "@codemirror/lang-python": "^6.2.1",
     "@codemirror/lang-sql": "^6.9.0",
+    "@codemirror/language": "^6.11.2",
+    "@codemirror/lint": "^6.8.5",
+    "@codemirror/search": "^6.5.11",
```

Comment:
We'll need to do global search anyways so I would just remove this, otherwise it's a little "inner" search area on the cell source.